react/features/conference/components/web/Conference.js

```diff
@@ -189,7 +190,7 @@ class Conference extends AbstractConference<Props, *> {
                 id = 'videoconference_page'
                 onMouseMove = { this._onShowToolbar }>
                 <Notice />
-                <Subject />
+                {SUBJECT_ALWAYS_HIDDEN || <Subject />}
```

Comment:
I think it makes more sense to hide it always when in filmstrip only mode, no need to the extra flag.